runtime/integration-tests/src/integrations_mock/mod.rs

```diff
@@ -41,25 +42,25 @@ cfg_if::cfg_if! {
             CollatorSelection, Council, DefaultBlocksPerRound, Democracy, EnactmentPeriod, RuntimeEvent,
             InflationInfo, LaunchPeriod, LeaveDelayRounds, NativeTokenExistentialDeposit, RuntimeOrigin,
             ParachainStaking, PolkadotXcm, Range, Runtime, Scheduler, Session, System,
-            TechnicalCommittee, Timestamp, TransactionPause, TransactionPayment, Treasury, Utility,
-            VotingPeriod, Preimage, ParachainSystem, ParachainInfo, AuraAuthorFilter, Aura, XcmpQueue,
-            CumulusXcm, DmpQueue, XTokens, Multisig, NonPausablePallets, AllPalletsWithSystem
+            TechnicalCommittee, Timestamp, TransactionPause, TransactionPayment, Treasury,
+            VotingPeriod, Preimage, NonPausablePallets, AllPalletsWithSystem
         };
         type RuntimeAssetConfig = calamari_runtime::assets_config::CalamariAssetConfig;
         type RuntimeConcreteFungibleLedger =
             calamari_runtime::assets_config::CalamariConcreteFungibleLedger;
     } else {
+        pub mod test_manta;
         use manta_runtime::{
-            assets_config::MantaConcreteFungibleLedger,
             currency::MANTA as KMA,
```

Comment:
oh i see the code sets MANTA = KMA lol

pls make it `UNIT` and add the alias for both chains

Reply:
https://github.com/Manta-Network/Manta/issues/1118 opened